src/components/standalone/firewall/conntrack/ConntrackContent.vue

```diff
@@ -0,0 +1,198 @@
+import ConntrackRecordsTable from '@/components/standalone/firewall/conntrack/ConntrackRecordsTable.vue'
+import DeleteConntrackRecordModal from '@/components/standalone/firewall/conntrack/DeleteConntrackRecordModal.vue'
+import {
+  NeHeading,
+  NeInlineNotification,
+  NeButton,
+  NeSkeleton,
+  NeEmptyState,
+  NeTextInput
+} from '@nethesis/vue-components'
+import { getAxiosErrorMessage } from '@nethesis/vue-components'
+import { onMounted, ref, computed } from 'vue'
+import { CanceledError } from 'axios'
+
+const { t } = useI18n()
+
+export type ConntrackRecord = {
+  destination: string
+  destination_port?: string
+  destination_stats: {
+    bytes: string
+    packets: string
+  }
+  id: string
+  protocol: string
+  source: string
+  source_port?: string
+  source_stats: {
+    bytes: string
+    packets: string
+  }
+  state?: string
+  timeout: string
+}
+
+const isloading = ref(false)
+const ConntrackRecordsResponse = ref<ConntrackRecord[]>([])
+const error = ref({
+  notificationDescription: '',
+  notificationDetails: '',
+  notificationTitle: ''
+})
+const filter = ref('')
+const showDeleteModal = ref(false)
+
+onMounted(() => {
+  fetchConntrack()
+})
+
+const selectedItem = ref<ConntrackRecord>()
+
+async function fetchConntrack() {
+  error.value.notificationDescription = ''
+  error.value.notificationDetails = ''
+  try {
+    isloading.value = true
+    ConntrackRecordsResponse.value = (await ubusCall('ns.conntrack', 'list')).data.data
+  } catch (err: any) {
+    if (!(err instanceof CanceledError)) {
+      error.value.notificationDescription = t(getAxiosErrorMessage(err))
+      error.value.notificationDetails = err.toString()
+    }
+  } finally {
+    isloading.value = false
+  }
+}
+function applyFilterToConntrackRecords(records: ConntrackRecord[], filter: string) {
+  const lowerCaseFilter = filter.toLowerCase()
+  return records.filter(
+    (ConntrackRecord) =>
+      ConntrackRecord.source.toLowerCase().includes(lowerCaseFilter) ||
+      ConntrackRecord.destination.toLowerCase().includes(lowerCaseFilter) ||
+      ConntrackRecord.protocol.toLowerCase().includes(lowerCaseFilter) ||
+      (ConntrackRecord.state ?? '').toLowerCase().includes(lowerCaseFilter) ||
+      (ConntrackRecord.source_port ?? '').toLowerCase().includes(lowerCaseFilter) ||
+      (ConntrackRecord.destination_port ?? '').toLowerCase().includes(lowerCaseFilter)
+  )
+}
+
+function delete_all() {
+  selectedItem.value = {
+    id: 'all',
+    destination: '',
+    destination_stats: { bytes: '', packets: '' },
+    protocol: '',
+    source: '',
+    source_stats: { bytes: '', packets: '' },
+    timeout: ''
+  }
+  showDeleteModal.value = true
+}
+
+function openDeleteModal(itemToDelete: ConntrackRecord) {
+  selectedItem.value = itemToDelete
+  showDeleteModal.value = true
+}
+function closeDeleteModal() {
+  showDeleteModal.value = false
+}
+
+async function reloadItems() {
+  await fetchConntrack()
```

Comment:
No need to `await` since there is nothing to do after `fetchConntrack()`. We could even remove `reloadItems` function and call `fetchConntrack()` directly